base/logger_file.go

```diff
@@ -287,7 +286,12 @@ func runLogDeletion(logDirectory string, logLevel string, sizeLimitMBLowWatermar
 	for i := len(files) - 1; i >= 0; i-- {
 		file := files[i]
 		if strings.HasPrefix(file.Name(), logFilePrefix+logLevel) && strings.HasSuffix(file.Name(), ".log.gz") {
-			totalSize += int(file.Size())
+			fi, err := file.Info()
+			if err != nil {
```

Comment:
Go docs say this error can occurr if the file doesn't exist any more - so I think the `continue` logic makes sense here, but would just like to log when it happens.

```suggestion
			if err != nil {
				InfofCtx(context.TODO(), "Couldn't get size of log file %q: %v - ignoring for cleanup calculation", file.Name(), err)
```